www/src/pages/ru/usage/env-variables.mdx

```diff
@@ -0,0 +1,124 @@
+
+```ts:env.mjs
+import { createEnv } from "@t3-oss/env-nextjs";
+import { z } from "zod";
+
+export const env = createEnv({
+  server: {
+    NODE_ENV: z.enum(["development", "test", "production"]),
+  },
+  client: {
+    // NEXT_PUBLIC_CLIENTVAR: z.string(),
+  },
+  runtimeEnv: {
+    NODE_ENV: process.env.NODE_ENV,
+  },
+});
+```
+
+T3 Env использует `createEnv`, который отвечает за создание схемы и будет включать главную логику валидации для клиентских и серверых переменных среды.
+
+<Callout type="info">
+  Для получения дополнительной информации о том, как `createEnv` работает изнутри, посмотрите документацию [T3
+  Env](https://env.t3.gg/docs/introduction) (На английском)
+</Callout>
+
+## Использование переменных среды
+
+Когда вы хотите использовать переменные среды, вы можете импортировать их из `env.mjs` и использовать их как вы обычно использовали бы. Если вы попробуете импортировать этот файл на стороне клиента и получить серверную переменную, вы получите ошибку:
+
+```ts:pages/api/hello.ts
+import { env } from "../../env.mjs";
+
+// `env` полностью типобезопасно и предоставляет автокомплит
+const dbUrl = env.DATABASE_URL;
+```
+
+```ts:pages/index.tsx
+import { env } from "../env.mjs";
+
+// ❌ Это даст ошибку
+const dbUrl = env.DATABASE_URL;
+
+// ✅ Тут все ОК
+const wsKey = env.NEXT_PUBLIC_WS_KEY;
+```
+
+## .env.example
+
+Изза того, что файл `.env` по умолчанию не добавляется в систему контроля версий, мы также добавили файл `.env.example`, в котором вы можете по желанию сохранить копию вашего файла `.env` с удаленными секретами. Это не обязательно, но мы рекомендуем держать пример в актуальном состоянии, чтобы сделать процесс настройки среды для новых участников проекта как можно проще.
+
+Некоторые фреймворки и инструменты сборки, такие как Next.js, предлагают хранить секреты в файле `.env.local` и коммитить файлы `.env` в ваш проект. Это не рекомендуется, поскольку это может облегчить случайный коммит секретов в ваш проект. Вместо этого мы рекомендуем хранить секреты в `.env`, держать ваш файл `.env` в `.gitignore` и коммитить только файлы `.env.example` в ваш проект.
+
+## Добавление переменных среды
+
+Для того, чтобы убедиться, что ваша сборка никогда не завершится без переменных среды, которые проект требует, вам нужно добавить новые переменные среды в **двух** местах:
+
+📄 `.env`: Введите переменную среды, как обычно делаете в файле `.env`, т.е. `KEY=VALUE`
+
+📄 `env.mjs`: Добавьте соответствующую логику проверки для переменных среды, определив для каждой из них Zod схему внутри `createEnv`, например `KEY: z.string()`. Кроме этого, убедитесь в том, что вы деструкткрировали их в опции `runtimeEnv`, например `KEY: process.env.KEY`.
+
+<Callout type="info">
+  Зачем нужно деструктурировать переменные среды внутри `runtimeEnv`?
+  Это связано с тем как Next.js собирает переменные среды в некоторых рантаймах.
```

Comment:
```suggestion
  Это связано с тем, как Next.js собирает переменные среды в некоторых рантаймах.
```

Reply:
I'm also unsure how to translate "runtime" properly :(
If there are no suggestions - leave as-is.